extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -287,6 +318,11 @@ export class AmpAnalytics extends AMP.BaseElement {
           (this.config_['vars'] && this.config_['vars'][name]) || '');
       return val + argList;
     });
+
+    // Add any given extraUrlParams as query string params
+    if (this.config_['extraUrlParams']) {
+      request = addParamsToUrl(request, this.config_['extraUrlParams']);
```

Comment:
I'll add to the docs.  In fact, this PR doesn't include any docs changes, so I'll need to fix that too :)